iphone/Maps/main.mm

```diff
@@ -3,11 +3,17 @@
 #include "platform/platform.hpp"
 #include "platform/settings.hpp"
 
+#include <clocale>
+
 int main(int argc, char * argv[])
 {
   auto & p = GetPlatform();
   LOG(LINFO, ("Organic Maps", p.Version(), "started, detected CPU cores:", p.CpuCores()));
 
+#ifdef OMIM_OS_IPHONE
```

Comment:
Doesn't it work on a mac? If it doesn't, then ifdef should also be above near the include. If it does, then ifdef is not needed.

Reply:
I first assumed that the `std::locale::global(std::locale(""));` call in `main.mm` configured the locale, so certain distance values were displayed with local decimal separator, for example in the place info panel:

![place_info](https://user-images.githubusercontent.com/25888296/219958694-dee50bed-0bc4-4d08-b70b-7a875ef4d278.png)

But now I've realized that these values are correctly displayed with the locale decimal separator because of the use of [MKDistanceFormatter](https://developer.apple.com/documentation/mapkit/mkdistanceformatter) that applies the localized decimal separator.

So this call to `std::locale::global(std::locale(""));` in `main.mm` can be deleted, as it doesn't make any use.